### PR TITLE
Change package name for module ams

### PR DESCRIPTION
**Pull request by zhoujinsong**
## Why are the changes needed?

Change package name to `org.apache.amoro` for module ams

## Brief change log

- Change package name to `org.apache.amoro` for module ams
- Change related docs and configuration files.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Review by XBaith — APPROVED** [review]
LGTM :)

**Review by tcodehuber — APPROVED** [review]

File: docs/admin-guides/managing-optimizers.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ You can submit optimizer in your own Flink task development platform or local Fl
  -Dtaskmanager.memory.managed.size=32mb \
  -Dtaskmanager.memory.network.max=32mb \
  -Dtaskmanager.memory.network.min=32mb \
- -c com.netease.arctic.optimizer.flink.FlinkOptimizer \
+ -c org.apache.amoro.flink.optimizer.FlinkOptimizer \
```
Comment (zhongqishang) [inline docs/admin-guides/managing-optimizers.md:262]:
```suggestion
 -c org.apache.amoro.optimizer.flink.FlinkOptimizer \
```

File: docs/admin-guides/managing-optimizers.md [hunk 2]
```diff
@@ -289,7 +289,7 @@ Or you can submit optimizer in your own Spark task development platform or local
  --conf "spark.driver.memory=g" \
  --conf "spark.executor.cores=1" \
  --conf "spark.executor.memory=2g" \
- --class com.netease.arctic.optimizer.spark.SparkOptimizer \
+ --class org.apache.amoro.spark.optimizer.SparkOptimizer \
```
Comment (zhongqishang) [inline docs/admin-guides/managing-optimizers.md:292]:
```suggestion
 --class org.apache.amoro.optimizer.spark.SparkOptimizer \
```

**Comment by zhoujinsong** [comment]
Thanks for the review. @XBaith @tcodehuber @zhongqishang 

I have made some changes due to your suggestion, please take another look. @zhongqishang

**Review by zhongqishang — APPROVED** [review]
LGTM.

**Pull request merged**